src/index.css

```diff
@@ -16,19 +16,26 @@
 .app__header__img {
   object-fit: contain;
   width: 150px;
+  
 }
 
 .app__header {
-  position: sticky;
+  position: fixed;
+  width: 100vw;
   top: 0;
   z-index: 1;
-  padding: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
   object-fit: contain;
   border-bottom: 1px solid var(--color);
   display: flex;
   justify-content: space-between;
   align-items: center;
   flex-wrap: wrap;
+
+}
+body {
```

Comment:
header is also a part of body so logically instead of giving padding to body, give app_posts the top padding

Reply:
I am getting a lot of errors, I don't think I have altered 'settings.json' file, but it showed this. I am not sure if I should follow the command shown or not..
![error-vscode](https://user-images.githubusercontent.com/86073340/223298124-6a154205-9f5e-40c8-93db-df307adc372f.png)

Reply:
I think I did it, Please check once..

Reply:
It's showing that there are conflicts in package.json and package-lock.json files, I haven't altered those files at all. I don't understand why it's showing conflicts in those

Reply:
Okay I'll look into it now

Reply:
Did u resolve it, shall I try again to merge all my 11 commits into one ?

Reply:
yes you can! :)

Reply:
Or you know what?,if you're facing problems, you can create a new branch from latest master and put these changes in it and create a pr from that branch only.

Reply:
u mean i should create a new branch to merge my commits into single one ?

Reply:
> u mean i should create a new branch to merge my commits into single one ?

I mean you should create a new branch from the latest master and fix the sticky navbar issue there. This branch's commit history is quite messed up

Reply:
> > u mean i should create a new branch to merge my commits into single one ?
> 
> I mean you should create a new branch from the latest master and fix the sticky navbar issue there. This branch's commit history is quite messed up so nuke this branch out of existence

Reply:
Sir, I tried to do what u suggested, but I don't know why I am continously getting conflicts in readme and package.json files, I haven't modified those files at all. Please tell me what to do.

Reply:
Shall I delete all newly-made branches and start fresh ??

Reply:
> Shall I delete all newly-made branches and start fresh ??

Yes, don't stress too much on this issue.

Reply:
> Sir, I tried to do what u suggested, but I don't know why I am continously getting conflicts in readme and package.json files, I haven't modified those files at all. Please tell me what to do.

They're arriving because of the merge commits in between (in those merge commits there're different changes and thus they're conflicting in the rebase flow of chronological order) , that's why I am suggesting to remove this branch.

Reply:
I have deleted this branch, and tried to update the changes related to sticky navbar in a new branch, but I see that the css file is already updated with the required changes, there's nothing left to modify.. what do I do now ?

Reply:
are you sure, css file is already updated? or have you missed something in using the correct branch? Also, if you can run the application, check if it's working fine and you're up to date with the website in production. As a final resort, delete the folder and clone the repository again, then create the branch.